packages/base/test/specs/ConfigurationChange.spec.js

```diff
@@ -0,0 +1,32 @@
+const assert = require("chai").assert;
+
+describe("Some configuration options can be changed at runtime", () => {
+	browser.url("http://localhost:9191/test-resources/pages/Configuration.html");
+
+	it("Tests that compactSize can be changed", () => {
+		const res = browser.execute( () => {
+			const config = window['sap-ui-webcomponents-bundle'].configuration;
+			config.setCompactSize(true);
+			return config.getCompactSize();
+		});
+		assert.strictEqual(res, true, "compactSize changed to true");
+	});
+
+	it("Tests that theme can be changed", () => {
+		const res = browser.executeAsync( async (done) => {
+			const config = window['sap-ui-webcomponents-bundle'].configuration;
+			await config.setTheme('sap_belize_hcb');
```

Comment:
When setting some value in tests and then checking it, if the value is set and read from a variable it is less error prone IMO. Especially for strings.
In this case:
```js
const newTheme = "sap_belize_hcb";
await config.setTheme(newTheme);
...
assert.strictEqual(res, newTheme, "Thems changed to HCB");
```

Reply:
I tried it but unfortunately browser.execute/executeAsync seems to run in a different scope and you can't access variables from the outside, so if you define newTheme in the score where the assert is, it won't be visible inside browser.execute.

Reply:
You can pass it in the context of browser.execute/executeAsync like this:
```js
const newTheme = "sap_belize_hcb";
browser.executeAsync( async (hcbTheme, done) => {
    await config.setThem(hcbTheme);
...
}, newTheme)
```

[Reference](https://webdriver.io/docs/api/browser/executeAsync.html)

Reply:
Cool! Thanks!